stable/rabbitmq/templates/deployment.yaml

```diff
@@ -42,6 +42,8 @@ spec:
           value: {{ default "/" .Values.rabbitmqVhost | quote }}
         - name: RABBITMQ_MANAGER_PORT_NUMBER
           value: {{ default "15672" .Values.rabbitmqManagerPort | quote }}
+        - name: RABBITMQ_DISK_FREE_LIMIT
+          value: {{ default "{mem_relative, 1.0}" .Values.rabbitmqDiskFreeLimit | quote }}
```

Comment:
Specifying a default here is unnecessary since the default is in `values.yaml`. Since this matches the existing style, I'd be fine with it. However, the default specified here is different to that in `values.yaml`. Why is this? It doesn't seem to make sense to me.

Reply:
The default here is the [bitnami default](https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables). The values.yaml has a sane default for the chart, which will prevent deployment errors.

Reply:
I meant there's `"{mem_relative, 1.0}"` vs. `'"8GiB"'` in `values.yaml` for the same value. This doesn't make sense to me.

Reply:
Maybe it makes sense, if you look at it this way:
`rabbitmqDiskFreeLimit` is tightly linked to the persistence volume.

I had to set the default to `'"8GiB"'` because of the default size of the persistence volume in the values.yaml (8Gi ). `'"8GiB"'` in the template, however, would look random, because persistence is left generic at this stage. Therefore, I think it makes more sense to set the default in the template to the default value defined by the bitnami container, which is `{mem_relative, 1.0}`.

Reply:
Why would it look random? `'"8GiB"'` is what is used. `{mem_relative, 1.0}` is just redundant and not used because the default comes from `values.yaml`. Actually, for new charts we recommend not to use the `default` function at all if possible in favor of defaults in `values.yaml`.

Reply:
Indeed, defining defaults at multiple points is a source for confusion.
My point was to stick to the container default, if you unset `rabbitmqDiskFreeLimit`, but I don't insist on that.